global_variables

```diff
@@ -1,5 +1,5 @@
 IMAGE="registry.fedoraproject.org/fedora:28"
-PKI_VERSION=10.6
-COPR_REPO="@pki/${PKI_VERSION}-nightly"
+COPR_REPO="@pki/10.6"
```

Comment:
This variable will collide with https://github.com/dogtagpki/pki/blob/master/travis/global_variables#L6 

Probably reuse from the `pki` repo?

Reply:
I take my comment back. Just realized other components (nuxwdog, jss, ...) will  need it